ampcrowd/basecrowd/urls.py

```diff
@@ -2,9 +2,10 @@
 
 from basecrowd import views
 
-urlpatterns = patterns('',
+urlpatterns = patterns(
```

Comment:
Ya, I'me not familiar with "django style", only what is in PEP8. Once we add tests, I would like to add Travis CI for testing/linting so making PEP8 changes as I go to prevent having to do it in a massive sweep.